qualitycontrol.sh

```diff
@@ -122,6 +122,7 @@ prepend-path PATH \$QUALITYCONTROL_ROOT/bin
 prepend-path LD_LIBRARY_PATH \$QUALITYCONTROL_ROOT/lib
 prepend-path LD_LIBRARY_PATH \$QUALITYCONTROL_ROOT/lib64
 prepend-path ROOT_INCLUDE_PATH \$QUALITYCONTROL_ROOT/include
+prepend-path ROOT_DYN_PATH \$QUALITYCONTROL_ROOT/lib
```

Comment:
Why not lib64?

Reply:
no idea :) I have just followed what was done in aliroot and aliphysics. If it should be lib64, we should modify them there as well.